scripts/coverage_growth.py

```
@@ -21,224 +23,292 @@
+        "--unified=0",
+        "--no-color",
+        "--no-ext-diff",
         "--no-renames",
+        "--src-prefix=a/",
+        "--dst-prefix=b/",
         f"{base_revision}...HEAD",
         "--",
     ]
     try:
-        completed_process = subprocess.run(
+        return subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)